MainApp/.gitignore

```diff
@@ -8,4 +8,4 @@ tmp/
 !.gitkeep
 dist
 .idea/
-public/downloads/*
```

Comment:
There was this bug occurring due to timeout function in gamePageCtrl.js not exiting. It was fixed by @shahsyedzafar  in this  commit https://github.com/yousuf031991/HumanAgentInteraction/pull/46/commits/7db7ceb3c386eda0000eee154522d4e61f1cf411.